fsck.ocfs2/journal.c

```diff
@@ -1068,12 +1074,63 @@ static errcode_t ocfs2_clear_journal_flag(ocfs2_filesys *fs,
 	return ret;
 }
 
+static errcode_t ocfs2_clear_jbd2_errno(ocfs2_filesys *fs,
+				struct ocfs2_dinode *di,
+				int slot)
+{
+	errcode_t ret = 0;
+	struct journal_info ji;
+	journal_superblock_t *jsb;
+
+	ji.ji_slot = slot;
+	ret = prep_journal_info(fs, slot, &ji, 0, 1);
+	if (ret) {
+		printf("Slot %d seems to have a corrupt journal.\n", slot);
+		goto bail;
+	}
+
+	jsb = ji.ji_jsb;
+	if (!jsb->s_errno)
+		goto bail;
+
+	printf("Clear jbd2 super block errno (%d) on slot %d\n",
+		jsb->s_errno, slot);
```

Comment:
Print the log once we successfully write journal superblock.

Reply:
ok